platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/copy/NodeToCopy.java

```diff
@@ -101,6 +104,10 @@ public MerkleNode getNodeToCopy() {
 
     @Override
     public String toString() {
-        return "{parent: " + newParent + ", index: " + indexInParent + ", node to copy: " + nodeToCopy + "}";
+        return new ToStringBuilder(this, JSON_STYLE)
```

Comment:
Does it make sense to use SHORT_PREFIX_STYLE everywhere? Why are a few places like this different and use JSON?

Reply:
Most of these `toString()` methods (including this one, I wrote this class 😸 ) are mostly just for debugging purposes. In general it is ok to change the style, e.g. not using the json style.